### PR TITLE
add howto page

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

**Comment by j0k3r** [comment]
Could you add a simple test that get the page and check the returned status ?

**Comment by nicosomb** [comment]
Done.

File: src/Wallabag/CoreBundle/Tests/Controller/StaticControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Wallabag\CoreBundle\Tests\Controller;
+
+use Wallabag\CoreBundle\Tests\WallabagCoreTestCase;
+use Doctrine\ORM\AbstractQuery;
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Tests/Controller/StaticControllerTest.php]: You don't need this line

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Tests/Controller/StaticControllerTest.php]: grrr, sorry for that. Fixed.

**Pull request merged**